### PR TITLE
add text to Market for eggs, etc to explain to new users that the items will also drop

### PR DESCRIPTION
**Pull request by Alys**
**DO NOT MERGE** until approved by @lemoness 

We often have new users in the Tavern ask (or complain) if there's any way to get eggs, hatching potions, or food without buying them. Before they reach level 4, they have no easy way to know that these items will drop, so it looks to them like the items are purchase-only.

This PR adds text to the Market under the headings for eggs, potions, and food, to explain that they items will also drop on task completion. See screenshot below. The text was chosen by @lemoness in https://plus.google.com/u/0/107946115700776803839/posts/5vt5vh4ifeC (private G+ group). The "learn more" link points to http://habitrpg.wikia.com/wiki/Drops

The screenshot has been hacked apart a bit to take up less space (e.g., rows of eggs deleted). The only change this PR makes is to add the text under the headings.

screen shot 2015-04-03 at 10 54 17 pm

### CODE REVIEW

File: common/locales/en/pets.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "inventoryText": "Click an egg to see usable potions highlighted in green and then click one of the highlighted potions to hatch your pet. If no potions are highlighted, click that egg again to deselect it, and instead click a potion first to have the usable eggs highlighted. You can also sell unwanted drops to Alexander the Merchant.",
     "food": "Food and Saddles",
     "noFood": "You don't have any food or saddles.",
+    "dropsExplanation": "Get these items faster with gems if you don't want to wait for them to drop when completing a task. <a href=\"http://habitrpg.wikia.com/wiki/Drops\">Learn more.</a>",
```
Comment (crookedneighbor) [inline common/locales/en/pets.json]: I think it should say "Learn more about the drop system" to be clear that the link is pointing to the wiki article about the drop system, and not an article about buying things with gems.

Comment (Alys) [inline common/locales/en/pets.json]: Agreed. I'll change it in a bit, after I see if there's any other initial feedback.

**Comment by Alys** [comment]
For Alys to do:

- [ ] Change "Learn more" to "Learn more about the drop system."

**Comment by gisikw** [comment]
Really love that this change is happening - this confused the heck out of me when I was getting started :)

**Comment by Alys** [comment]
@gisikw Good to know! (I mean that in the nicest possible way.) What do you think of the wording? Would it have made it clear to you that buying eggs etc was not necessary?

**Comment by gisikw** [comment]
Yep, definitely makes sense! :)

**Comment by gisikw** [comment]
Though it might be worth making the drop system modal (`firstDrop`) more explicit as well? Something like:

"You've unlocked the Drop System! Now when you complete tasks, you have a small chance of finding an item _(eggs, potions, food, and maybe more!)_. You just found a <strong><%= eggText %> Egg</strong>! <%= eggNotes %>"

**Comment by Alys** [comment]
_(Grrraarrrr. Should not have been closed. It must have happened automatically when I accidentally deleted my branch.)_

@lemoness What do you think of this?

#### For Alys to do, from comments above:
- [ ] Change "Learn more" to "Learn more about the drop system."
- [ ] Do something about [this comment](https://github.com/HabitRPG/habitrpg/pull/4969#issuecomment-89295130) re drop system modal (depending on any future comments that might be made)

**Comment by lemoness** [comment]
The modal change looks good to me (although note that it will always say
"wolf egg").

**Comment by Alys** [comment]
@lemoness Final changes for your approval:

screen shot 2015-04-14 at 7 49 19 am

.

screen shot 2015-04-14 at 7 59 48 am

**Comment by lemoness** [comment]
Options > Inventory? That's very outdated. Should be Inventory > Market!

Also, "maybe more!)." has some awkward punctuation. And what else do we get
aside from eggs, potions, and food? Quest scrolls only LOOK like they drop;
they're actually level-based.

I'd say "chance of finding an item, including eggs, potions, and food!"

**Comment by crookedneighbor** [comment]
:+1: to both points ^

Could we make the market text a link?

**Comment by Alys** [comment]
I was wondering about the "maybe more" text.

@lemoness Do you want a link in the modal? My preference is usually to force the users to use the menu system so that the next time they want to get there, when the link is no longer in front of them, they have a better chance of remembering how they got there previously.

**Comment by lemoness** [comment]
I don't have a strong preference either way. We can start without a link
and see if it causes confusion.

**Comment by crookedneighbor** [comment]
@Alys that's a good point.

**Comment by Alys** [comment]
@lemoness I've made those changes and merged this. No one complained about "Options > Inventory" being wrong for months, so I doubt they'll complain about the lack of a link. :-/   If more people would read this modal, we'd get fewer questions about how to hatch an egg. :-[

**Comment by lemoness** [comment]
"If more people would read..." is the beginning of so many customer service
laments ;p

Thanks!

**Pull request merged**